config/optional/amp.theme.yml

```diff
@@ -0,0 +1 @@
+amptheme: thunder_amp
```

Comment:
Do you think, we really need this as an optional config? We could let it be in the themes_installed hook. At least we have everything in the same spot then.